contracts/Main.sol

```diff
@@ -65,25 +64,49 @@ contract Main {
         newTRBContract.mint(msg.sender, balance);
     }
 
-    // Helper function for contracts with public admin to
-    // send the funds directly to its address.
-    //slither-disable-next-line unimplemented-functions
-    function migrateContract(address _contract) public onlyAdmin {
```

Comment:
I removed this function because I thought it opened up the possibility of a few weird edge cases, without increasing that much usability.